content/dsl_recipe.md

```diff
@@ -391,7 +390,7 @@ find_resource(:template, '/etc/seapower') do
 end
 ```
 
-## find_resource!
+## find_resource
```

Comment:
There are a few changes like this in this PR that don't seem correct. `find_resource!` and `platform?` have punctuation as part of their method name so removing that could be confusing.

Reply:
Good point! Let me revert & then pick through the changes carefully.